topics/assembly/tutorials/chloroplast-assembly/tutorial.md

```diff
@@ -0,0 +1,312 @@
+---
+layout: tutorial_hands_on
+
+title: "Chloroplast genome assembly"
+zenodo_link: "http://doi.org/10.5281/zenodo.3567224"
+questions:
+  - "How can we assemble a chloroplast genome?"
+objectives:
+  - "Assemble a chloroplast genome from long reads"
+  - "Polish the assembly with short reads"
+  - "Map reads to the assembly and view"
+  - "View an annotated assembly"
+time_estimation: "2h"
+key_points:
+  - "A chloroplast genome can be assembled with long reads and polished with short reads"
+  - "The assembly graph is useful to look at and think about genomic structure"
+  - "We can map raw reads back to the assembly and investigate areas of high or low read coverage"
+  - "We can view an assembly, its mapped reads, and its annotations in JBrowse"
+contributors:
+  - annasyme
+---
+
+# Introduction
+{:.no_toc}
+
+## What is genome assembly?
+
+Genome assembly is the process of joining together DNA sequencing fragments into longer pieces, ideally up to chromosome lengths.The DNA fragments are produced by DNA sequencing machines, and are called "reads". These are in lengths of about 150 nucleotides (base pairs), to up to a million+ nucleotides, depending on the sequencing technology used. Currently, most reads are from Illumina (short), PacBio (long) or Oxford Nanopore (long and extra-long).
+
+It is difficult to assemble plant genomes as they are often large (for example, 3,000,000,000 base pairs), have many repeat regions (such as transposons), and may be polyploid. This tutorial shows genome assembly for a smaller data set - the plant chloroplast genome - a single circular chromosome which is typically about 160,000 base pairs. We will use a subset of a real data set from sweet potato, from the paper {% cite Zhou2018 %}.
+
+>### Agenda
+> In this tutorial we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Upload data
+
+Let's start with uploading the data.
+
+> ### {% icon hands_on %} Hands-on: Import the data
+> 1. Create a new history for this tutorial and give it a proper name
+>
+>    {% include snippets/create_new_history.md %}
+>    {% include snippets/rename_history.md %}
+>
+> 2. Import from [Zenodo](https://zenodo.org/record/3567224) or a data library (ask your instructor):
+>   - FASTQ file with illumina reads: `sweet-potato-chloroplast-illumina-reduced.fastq`
+>   - FASTQ file with nanopore reads: `sweet-potato-chloroplast-nanopore-reduced.fastq`
+>   - Note: make sure to import the files with "reduced" in the names, not the ones with "tiny" in the names.
+>    ```
+>    https://zenodo.org/record/3567224/files/sweet-potato-chloroplast-illumina-reduced.fastq
+>    https://zenodo.org/record/3567224/files/sweet-potato-chloroplast-nanopore-reduced.fastq    
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+{: .hands_on}
+
+# Check read quality
+
+We will look at the quality of the nanopore reads.
```

Comment:
These reads are high quality so don't need filtering but the Q&A section talks about what sorts of things are useful to consider. I want to avoid a tutorial that is too long so have not covered QC in detail here but I will point to other tutorials that cover it in more detail.